src/components/Personalization/actions/insertBefore.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { showElements } from "../flicker";
+
+const DIV_TAG = "DIV";
+
+const createFragment = content => {
+  const result = document.createElement(DIV_TAG);
+  result.innerHTML = content;
+
+  return result;
+};
+
+const insertBefore = (container, content) => {
+  const fragment = createFragment(content);
+  const elements = [].slice.call(fragment.children);
```

Comment:
I've seen `[].slice.call(` a couple times now. I think it should be moved into a utility whenever you go through the process of extracting common code.

Reply:
Agreed, will do.